src-vue/src/components/PlayButton.vue

```diff
@@ -53,17 +85,53 @@ export default defineComponent({
     <el-button :disabled="northstarIsRunning" type="primary" size="large" @click="launchGame" class="fc_launch__button">
         {{ playButtonLabel }}
     </el-button>
+    <el-select v-model="currentCanal" placeholder="Select">
+        <el-option-group
+            v-for="group in selectOptions"
+            :key="group.label"
+            :label="group.label"
+        >
+            <el-option
+                v-for="item in group.options"
+                :key="item.value"
+                :label="item.label"
+                :value="item.value"
+            />
+        </el-option-group>
+    </el-select>
```

Comment:
Maybe we should hook this up to a button in settings so by default it's hidden but clicking "Enable testing release channels" will cause the button to show. Just so that someone just starting out on Northstar doesn't end up on release candidate by accident ^^

And then we should also persist to disk if the dropdown button to switch release channel was pressed.
(I can do the persisting to disk part if you want to)

Reply:
Should I replace the "Toggle release candidate" button by this "Enable testing release channels" button?

Reply:
Maybe a checkbox instead of a button so that the state on/off is clear but yeah, something like that would be great ^^

Reply:
Added in ff915f21d9a69caf02844aee997810b8e275aa63